### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by Meihan-Liu** [Skyorca/Awesome-Graph-Domain-Adaptation-Papers#1]
SOGA 是WSDM24的文章

**Comment by Skyorca** [comment]
You are right. At the time of paper collection&survey writing, SOGA has not been published yet. In fact it has appeared on Arxiv for years until being accepted
